### DISCUSSION TITLE
how we could remove the legend

### DISCUSSION THREAD

**Discussion by squigglybob** [Pray4Movement/prayer-global-porch#39] [Ideas]
So I've been thinking about how we could remove the need for the legend...
In my mind everything in the bottom section of the map *is* the legend.

Currently
![image](https://user-images.githubusercontent.com/9816214/201876754-8519b65b-aaba-47ae-ad5d-4ee0c20a9abe.png)

So we could emphasize the colours of the green and red in the foot block by reversing the colours of the numbers with the background like so...
![image](https://user-images.githubusercontent.com/9816214/201872361-239d7116-90f3-4feb-9be7-ce13407e9c6c.png)

would that make it more obvious that these are the colours represented on the map?
Or should we also make it more legendy like the legend I added.

Maybe something like this?
![image](https://user-images.githubusercontent.com/9816214/201876481-b3005d92-2efe-4cd2-9726-574679afdb83.png)

@ChrisChasm @corsacca @himayrunner @ahillbilly @prykon

**Comment by corsacca** [comment]
![image](https://user-images.githubusercontent.com/24901539/201880458-0f7f0b55-af0c-41ee-8605-99b526b131f8.png)
It might work to  add the color box and leave them empty by the label

**Reply by squigglybob** [reply]
as in put the colour swatch to the left of the words, and decolourise the numbers?
